api/src/infrastructure/data-source-manager.ts

```diff
@@ -1,24 +1,57 @@
 import * as fs from 'fs';
-import { Injectable, Logger } from '@nestjs/common';
+import { Inject, Injectable, Logger } from '@nestjs/common';
 import { InjectDataSource } from '@nestjs/typeorm';
 import { DataSource } from 'typeorm';
-import { SectionsCSVParser } from 'api/data/sections-csv-parser';
-import { SQLAdapter } from 'api/data/sql-adapter';
+import { SectionsCSVParser } from 'api/data/sections/sections-csv-parser';
+import { SQLAdapter } from '@api/infrastructure/sql-adapter';
+import { PageFiltersCSVParser } from 'api/data/page-filters/page-filters-csv-parser';
 
 @Injectable()
 export class DataSourceManager {
   public constructor(
     private readonly logger: Logger,
+    @Inject(SQLAdapter) private readonly sqlAdapter: SQLAdapter,
     @InjectDataSource() private readonly dataSource: DataSource,
   ) {}
 
   public async loadInitialData(): Promise<void> {
+    const schemaSql = this.getInitialSchemaSqlCode();
```

Comment:
There is a redundant call to this method

Reply:
Removed